.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
You should remove this file, the .eslintrc.json file is all you need for this assignment